deploy/updater/updater.py

```
@@ -4,329 +4,397 @@
-            drones_v1 = []
-            drones_svg = []
-
-            try:
-                for region_name in regions:
-                    time_diff = await calculate_time_difference(
-                        alerts_data["states"][region_name]["changed"], current_datetime
+                logger.debug("alerts_websocket_v1 not changed")
+
+        except Exception as e:
+            logger.error(f"update_alerts_websocket_v1: {str(e)}")
+            logger.debug(f"Повний стек помилки:", exc_info=True)
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)